go/common/node/node.go

```diff
@@ -109,6 +112,15 @@ const (
 	// RoleReserved are all the bits of the Oasis node roles bitmask
 	// that are reserved and must not be used.
 	RoleReserved RolesMask = ((1 << 32) - 1) & ^((RoleStorageRPC << 1) - 1)
+
+	RoleComputeWorkerName = "compute"
```

Comment:
Add a doc comment for this variable block.

Reply:
Done.